### ISSUE TITLE
why label this file only have camera_extrinsics.npy 

### ISSUE THREAD

**Issue by heiheiwangergou** [foocker/lsp#12]
when i bash run.sh   ，until  200steps ,  the code is finsh ,no error ,but in label file only have  camera_extrinsics.npy ，no other npy file

**Comment by tylersky1993** [comment]
The same thing is happening to me!

**Comment by heiheiwangergou** [comment]
> The same thing is happening to me!

do you have created your person video ?
it dose work ?

**Comment by tylersky1993** [comment]
> 

No, I stopped there

**Comment by tylersky1993** [comment]
> > The same thing is happening to me!
> 
> do you have created your person video ? it dose work ?

No, I stopped there

**Comment by foocker** [comment]
> when i bash run.sh ，until 200steps , the code is finsh ,no error ,but in label file only have camera_extrinsics.npy ，no other npy file


> when i bash run.sh ，until 200steps , the code is finsh ,no error ,but in label file only have camera_extrinsics.npy ，no other npy file

sorry, I annotated the complete code for testing。 in lsp_data_generator.py, you should:
```if i <= 200:  # remove this line
                img = plot_kpts(img, choosed_landmark)
                cv2.imwrite(f'./out/kh_orth_proj_3d_rot_trans_2d_{i}.png', img)  # change to your own save dir/name

# and  not annotate all of these line:
  # np.save(os.path.join(savefolder, 'img_meta.npy'), img_meta)
  # f = lambda x: torch.stack(x, dim=0).cpu().numpy()
  # pts_3d, rot_angles, trans = f(pts_3d), f(rot_angles), np.expand_dims(f(trans), axis=-1)
  # pts_3d_224 = f(pts_3d_224)
  # mean_pts_3d = np.mean(pts_3d, axis=0)
  # # print(mean_pts_3d.shape, pts_3d.shape)
  # np.save(os.path.join(savefolder, 'mean_pts_3d.npy'), mean_pts_3d)
  # # print(pts_3d.shape, rot_angles.shape, trans.shape)
  # np.savez(os.path.join(savefolder, '3d_fit_data.npz'), pts_3d=pts_3d, rot_angles=rot_angles, trans=trans)  # TODO 
  
  # np.save(os.path.join(savefolder, 'landmarks2d_original.npy'), landmarks2d_original)
  # np.save(os.path.join(savefolder, 'tforms.npy'), tforms)
  # np.save(os.path.join(savefolder, 'cams.npy'), cams)
  
  # np.savez(os.path.join(savefolder, '3d_fit_data_224.npz'), pts_3d=pts_3d_224, rot_angles=rot_angles, trans=trans) 
```

**Comment by leung-yaya** [comment]
hi,@heiheiwangergou ,the train data is your own data or Author's data?
